src/tactic/core.lean

```diff
@@ -1325,13 +1329,41 @@ do e ← pformat_macro () s,
 
 reserve prefix `trace! `:100
 /--
-the combination of `pformat` and `fail`
+the combination of `pformat` and `trace`
 -/
 @[user_notation]
 meta def trace_macro (_ : parse $ tk "trace!") (s : string) : parser pexpr :=
 do e ← pformat_macro () s,
    pure ``((%%e : pformat) >>= trace)
 
+/-- an alias of `format` that gives tactics an expression to pretty pring -/
+@[reducible] meta def fmt_of {α} (x : α) : Type := format
```

Comment:
The declarations below using `fmt_of` seem to not be used anywhere else. What is their use case?